src/config/types.ts

```diff
@@ -161,4 +161,5 @@ export interface ConfigType {
   announcements: AnnouncementsEnabled | AnnouncementsDisabled;
   socialMediaLinks: SocialMediaLink[];
   unresolvedGeoIdErrorMessage?: UnresolvedGeoIdErrorMessage;
+  clockType: 12 | 24;
```

Comment:
This should be moved to `defaultConfig.settings` beside languages and other related info.  `defaultConfig.ts.example` should be updated too.

Reply:
Moved the clockType property under settings and added clockType to the example config.